developer_guide/installation/index.rst

```diff
@@ -11,11 +11,12 @@ Once the requirements are fulfilled, you are ready to install Akeneo PIM. Otherw
     installation_ee_archive
     setup_hybrid_storage_mysql_mongo
 
-Here are the guidelines to setup system requirements on specific operating systems:
+Here are the guidelines to set up your system on specific OS:
```

Comment:
I disagree, you'd have twice the word "system". I prefer "to set everything up according to your operating system".